src/components/organisms/UiMenu/stories/HasButton.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <UiPopover class="has-button">
+    <UiMenu
+      v-bind="args"
+      role="listbox"
+      :aria-label="ariaLabel"
+      aria-multiselectable="true"
+      :aria-activedescendant="ariaActivedescendant"
+      :aria-busy="ariaBusy"
+    >
+      <template #[`menu-item-${index}`] v-for="(item, index) in args.items">
+        <template v-if="item.isButton">
+          <UiButton
```

Comment:
this code will return <UiButton in UiButton I thing so. You should add content for this button